app/views/bus_stops/by_sequence.haml

```diff
@@ -1,13 +1,16 @@
 .page-title Route #{@route.number} Bus Stops
 
-.container
+%div
```

Comment:
Why do we want this link in a div?

Reply:
It was the simplest way to keep it out of line with the next text, other than a few `br` tags .